src/components/navbar/Navbar.jsx

```diff
@@ -27,21 +28,50 @@ export default function Navbar() {
               <i id="nav-search-icon" className="fas fa-search icon" />
             </span>
           </a>
-          <a href="">
-            <i className="fas fa-user" />
-          </a>
-          <Link to="/wishlist" element={<Wishlist />}>
-            <span className="icon-container badge-container">
-              <i className="fas fa-heart icon" />
-              <span className="icon-badge">2</span>
-            </span>
-          </Link>
-          <Link to="/cart" element={<CartManagement />}>
-            <span className="icon-container badge-container">
-              <i className="fas fa-shopping-cart icon" />
-              <span className="icon-badge">2</span>
-            </span>
-          </Link>
+
+          {authState.user ? (
+            <Link to="/user">
+              <span>
+                <i className="fa-solid fa-user-check" />
+              </span>
+            </Link>
+          ) : (
+            <Link to="/login">
+              <span>
+                <i className="fa-solid fa-user-xmark" />
+              </span>
+            </Link>
+          )}
+
+          {authState.user ? (
+            <Link to="/wishlist">
+              <span className="icon-container badge-container">
+                <i className="fas fa-heart icon" />
+                <span className="icon-badge">2</span>
+              </span>
+            </Link>
+          ) : (
+            <Link to="/login">
+              <span className="icon-container badge-container">
+                <i className="fas fa-heart icon" />
+              </span>
+            </Link>
+          )}
```

Comment:
Rather than having the ternary for two Links you could add it to the `to` porp only as all rest things are same in both the elements.
e.g
```jsx
<Link to={authState.user ? "/wishlist" : "/login"}>
  <span className="icon-container badge-container">
    <i className="fas fa-heart icon" />
    {authState.user
       ? <span className="icon-badge">2</span>
       : null
     }
   </span>
</Link>
```